Makefile

```diff
@@ -55,6 +56,10 @@ tests:
 
 all-tests: checks tests
 
+watch:
+	$(DOCKER_COMPOSE) run --rm --service-ports web /bin/bash -c \
```

Comment:
Actually why do you add `--service-ports` to those commands? Is that for the debugger? It prevents being able to run the service in the background and additionally run those commands.